### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by polarker**

### CODE REVIEW

File: packages/extension/src/ui/features/lock/ResetScreen.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ export const ResetScreen: FC = () => {
     >
       <P>
         If you reset your wallet, the only way to recover it is with your
-        12-word seed phrase. Make sure to back it up from the Argent X settings
```
Comment (h0ngcha0) [inline packages/extension/src/ui/features/lock/ResetScreen.tsx:28]: Do we need to update here?
https://github.com/alephium/extension-wallet/blob/polish/packages/extension/src/ui/features/recovery/seedRecovery.state.ts#L14

Comment (polarker) [reply packages/extension/src/ui/features/lock/ResetScreen.tsx:28]: We are still using 24 words. I created a PR for wallet SDK to support 12 words. Once that is merged and released, I will update to use 12 words

Comment (h0ngcha0) [reply packages/extension/src/ui/features/lock/ResetScreen.tsx:28]: Ok, thanks :+1:

**Review by h0ngcha0 — APPROVED** [review]
LGTM :+1:

**Pull request merged**